tests/bluetooth/tester/src/btp_gap.c

```diff
@@ -1305,6 +1315,8 @@ static uint8_t set_filter_list(const void *cmd, uint16_t cmd_len,
 		}
 	}
 
+	filter_list_in_use = !!cp->cnt;
```

Comment:
```suggestion
	filter_list_in_use = cp->cnt != 0;
```

I think this is more readable